library/platform_util.c

```diff
@@ -133,3 +134,14 @@ struct tm *mbedtls_platform_gmtime_r( const mbedtls_time_t *tt,
 #endif /* _WIN32 && !EFIX64 && !EFI32 */
 }
 #endif /* MBEDTLS_HAVE_TIME_DATE && MBEDTLS_PLATFORM_GMTIME_R_ALT */
+
+#if defined(MBEDTLS_CHECK_PARAMS) && defined(MBEDTLS_DEBUG_INVALID_PARAMS)
+void mbedtls_param_failed( const char* failure_condition, const char* file,
```

Comment:
I am not sure this is going to work. The problem is that if MBEDTLS_PARAM_FAILED is defined, the function mbedtls_param_failed() will be implemented here and somewhere in the user files.